content/docs/develop/reference/troubleshooting/index.md

```diff
@@ -37,6 +41,33 @@ In this section, instructions are summarized that help to trouble shoot known is
     ```
     kubectl delete pods --all -n knative-eventing
     ```
+</p></details>
+
+## Control Servic is not available
+
+### CLI command was not executed correctly
+<details><summary>Expand instructions</summary>
+<p>
+
+**Investigation:**
+
+The control service is not available at the time when a command was sent by the keptn CLI. 
+The resulting response message will look similar to this:
+
+    ```console
+    keptn onboard service --project=sockshop --values=values_carts.yaml
+    
+    Starting to onboard service
+    Onboard service was unsuccessful
+    Error: Post https://control.keptn.1xx.xxx.xx.xx.xip.io/service: dial tcp: lookup control.keptn.1xx.xxx.xx.xx.xip.io: no such host
+    ``` 
+
+**Reason:** 
+
+Probably, the cluster is overloaded
+
+**Solution:** 
 
+Please wait a couple of minutes for the cluster to recover and try again.
```

Comment:
Would more resources in the cluster help to solve this issue? Waiting for an arbitrary time is again somewhat vague. If we don't know how to solve a problem, then let's not do guesswork on what the solution could be. Either we know, or it's a known issue that we're working on figuring out.

Reply:
I created an issue [#392](https://github.com/keptn/keptn/issues/392), which targets this problem.